eth/backend.go

```diff
@@ -267,6 +267,7 @@ func New(
 		allowUnprotectedTxs:      config.AllowUnprotectedTxs,
 		allowUnprotectedTxHashes: allowUnprotectedTxHashes,
 		allowUnfinalizedQueries:  config.AllowUnfinalizedQueries,
+		stateQueryWindow:         config.HistoricalStateQueryWindow,
```

Comment:
nit: let's be consistent with names when we propagate something with a related name.